redis/README.md

```diff
@@ -49,7 +49,7 @@ $ docker run --name some-app --link some-redis:redis -d application-that-uses-re
 ## ... or via `redis-cli`
 
 ```console
-$ docker run -it --link some-redis:redis --rm redis sh -c 'exec redis-cli -h "$REDIS_PORT_6379_TCP_ADDR" -p "$REDIS_PORT_6379_TCP_PORT"'
+$ docker run -it --link some-redis:redis --rm redis sh -c 'exec redis-cli -h "redis" -p "6379"'
```

Comment:
:+1:; since we're not using the env variables anymore, we can drop the `sh -c` also and make this much simpler:
```console
$ docker run -it --link some-redis:redis --rm redis redis-cli -h redis
```

(also, `README.md` is generated, so if you could also move this change over to `content.md`, that'd be appreciated!)